src/Directory.Build.props

```diff
@@ -51,6 +50,6 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <NoWarn>1591;1572;1573;1574</NoWarn>
     <DocumentationFile></DocumentationFile>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Do we want his or is it a temporary change?

Reply:
There are way too many fixes that must be done, so we'll leave it like this for now so it compiles.

Reply:
ok fine so we can slowly fix warnings